akka-http-core/src/main/scala/akka/http/javadsl/ServerBinding.scala

```diff
@@ -112,6 +111,18 @@ class ServerBinding private[http] (delegate: akka.http.scaladsl.Http.ServerBindi
     delegate.whenTerminated
       .map(_.asInstanceOf[HttpTerminated])(ExecutionContexts.sameThreadExecutionContext)
       .toJava
+
+  /**
+   * Adds this `ServerBinding` to the actor system's coordinated shutdown, so that [[unbind]] and [[terminate]] get
+   * called appropriately before the connection pools are shut down.
```

Comment:
"before the connection pools are shut down"? I guess this should be rephrased "before the actor system is shut down (or terminated)".